app/ml.py

```diff
@@ -86,3 +87,34 @@ async def get_crime_score(city: str):
     ret_dict["msg"]     = f"{city} crime score"
     ret_dict["score"]   = gen_crime_score(city_scl[0])
     return ret_dict
+
+@router.get('/rent_rate/{city}')
+async def get_rent_rate(city: str):
+  '''
+  Takes in a city as its parameter and returns a dictionary of keys including:
+   - Dec Avg Rent
+   - Jan Proj Rent
+   - Feb Proj Rent
+   - Mar Proj Rent
+   Average last month's rent in that city and projected rent for the next three months
+  '''
+  cursor = db_conn.cursor(cursor_factory=RealDictCursor)
+  sql = f'SELECT * FROM cityspire_rent WHERE "City"= \'{city}\';'
```

Comment:
Hey... my understanding is that constructing a SQL statement by building/interpolating a string is not recommended because the characters you are inserting could be destructive to the database (city = `DROP my_db`)

You may want to look at: https://www.psycopg.org/docs/usage.html#query-parameters

![CleanShot 2021-02-02 at 13 33 10](https://user-images.githubusercontent.com/560998/106652878-b4855b80-655b-11eb-9c07-13036826dbf9.png)

It may be worth taking another look at these lines and updating just to get into a habit of using safe method.

I think I am using the appropriate method but need to check myself!